tsconfig.json

```diff
@@ -6,6 +6,12 @@
       "dom.iterable",
       "esnext"
     ],
+    "baseUrl": "src",
+    "paths": {
+      // You can do import {whatever} from 'Example/'
+      // and it would point inside of the Maps folder
+      "Example/*": ["Maps/*"]
```

Comment:
And here you can play around with whatever other aliases you want to create. But I think that it's enough with `baseUrl`